src/lib/service/messaging/index.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+export class MessageDispatcher {
+	public api: Function
+
+	constructor(callback: Function) {
+		this.api = callback
+	}
+}
+
+export function slack(uri: string): MessageDispatcher {
+	return {
+		api: (message: string) => {
+			axios.post(uri, {
+				text: message,
+			})
+		},
+	}
+}
+
+export function send(message: string, dispatcher: MessageDispatcher) {
```

Comment:
This is somewhat confusing. Could we at some point in the future, simplify to just having a `send(msg)` function which just uses the `config.BOOKING_ALERT_WEBHOOK` url directly? Thanks!

Reply:
Yeah, it's ended up a little more complicated than it needs to be for now as I mentioned to Alan wanting to be able to support multiple endpoints - e.g. have the slack message and the rpi pinged to flash some lights 🕺 🎉